retry.js

```diff
@@ -13,7 +13,9 @@ const retryDefaults = {
 const withRetrying = (createClient, retryOpts = {}) => {
 	retryOpts = Object.assign({}, retryDefaults, retryOpts)
 
-	const createRetryingClient = (profile, userAgent, request = _request) => {
+	const createRetryingClient = (profile, userAgent, opt = {}) => {
+		const {request} = Object.assign({request: _request}, opt)
```

Comment:
You could move this to a `defaults` variable to be consistent with `index.js` (or even move this into its own function, since it's used three times), but I don't mind if you don't.

Reply:
Labeling this object as `defaults` sounds like it represents all options that `createClient` from `index.js` knows about. All I want to do here get the `request` fn if it has been passed in.

Reply:
I could also change the line below to `const request = 'request' in opt ? opt.request : _request`.